CEDS-2027 movement details

MaciejRewera opened the pull request.

ed0906 commented on app/views/components/gds/inputDate.scala.html:
```diff
@@ -0,0 +1,66 @@
+@this(dateInput: GovukDateInput)
+
+@(
+  field: Field,
```
Looks like you just use this to get `field.name` maybe just pass that in instead?

ed0906 commented on app/views/components/gds/inputTime.scala.html:
```diff
@@ -0,0 +1,58 @@
+@this(dateInput: GovukDateInput)
+
+@(
+  field: Field,
```
Similar, looks like you just use this to get `field.name`. Maybe just pass that in instead?

MaciejRewera commented:
I've addressed you comment. This was mostly copied from the previous implementation. But it turned out to be better after changed proposed.
Had to force-push as I've removed a commit from thte history.

MaciejRewera commented:
Another force-push, as I did not rebase.
